WebRoot/skins/bare/manifest.xml

```diff
@@ -44,5 +44,14 @@
 			<file>../../css/zhtml.css</file>
 			<file>../_base/base3/zhtml.css</file>
 		</css>
-	</standard>		
+	</standard>
+	<print>
+		<css>
+			<file>../_base/base3/print.css</file>
+		</css>
+		<cssIgnore>
```

Comment:
we are including skin.css in https://github.com/Zimbra/zm-web-client/pull/788/files#diff-d34b0fa9ff6036a11669d2a34ee3404cc6853a6b851e5932af54a0610d7e2ae6R47 and then ignoring here, looks a bit weird

Reply:
Previously I tried another approach not to load `../_base/base3/skin.css` when `client=print` is specified.
* https://github.com/Zimbra/zm-web-client/pull/769

However, it broke another pages. There are some pages which request skin.css without specifying client type. In this case, only `common` block is checked. No css was loaded with PR 769.

We need to keep `common` block as it is to avoid regression on other pages.
The purpose of the change is to load only print.css in a skin. So we need to remove css file(s) added in `common` block.